scripts/main.js

```diff
@@ -0,0 +1,18 @@
+import { guestList } from "./Guests.js"
+const mainContainer = document.querySelector("#container")
+
+const applicationHTML = `
+<section class="services"
+
+</section>
+<div class="areas_and_guests">
+<section class="areas"
+
+</section>
+<aside class="guestlist">
+${guestList()}
+</section>
```

Comment:
I think this should be `</aside>` instead of a closing section tag

Reply:
good eye